### PR TITLE
fix(metadata): hide google

### LINKED ISSUE #2534: Labeling: Fixes
- [ ] Rename Account – **Cursor is not visible** 

- [ ] **Remove the original name** on both Account and Wallet on hover – It's irrelevant, user can still see it when deleting the label. 

- [ ] Rename Wallet – Apply **the same design principle** (**[Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5f3bf73d694a9912ac30606f)**) as renaming the Accounts. Use the tag `Rename`, get rid of the edit menu, since clicking on the wallet name is not intuitive and may load the wallet itself as opposed to opening the menu.

### PR DESCRIPTION
**Pull request by mroz22**
I am still going to do some cleaning up here.. so aaalmost ready to review.

changes: 
- implement better UI for labeling
- ditch google
- rework integration tests to work with dropbox
- hopefully fix error with unexpected toasts. I supspect it might have been caused by improperly handled  intervals when working with multiple wallets

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/plugins/dropbox.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
+const express = require('express');
+var bodyParser = require('body-parser');
```
Comment (keraf) [inline packages/integration-tests/projects/suite-web/plugins/dropbox.js]: I don't really care about the coding style in the tests much but `var`?

File: packages/integration-tests/projects/suite-web/tests/metadata/dropbox.test.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { rerouteDropbox, stubOpen } from '../../stubs/metadata';
+
+// probably not needed
+describe.skip('Metadata', () => {
+    beforeEach(() => {
+        cy.viewport(1024, 768).resetDb();
+    });
+
+    after(() => {
+        // cy.task('stopGoogle');
```
Comment (keraf) [inline packages/integration-tests/projects/suite-web/tests/metadata/dropbox.test.ts]: No `stopDropbox` here?

File: packages/suite/src/actions/suite/metadataActions.ts [hunk 3]
```diff
@@ -141,9 +142,12 @@ export const disconnectProvider = () => async (dispatch: Dispatch) => {
 const handleProviderError = (error: MetadataProviderError, action: string) => (
     dispatch: Dispatch,
 ) => {
+    console.log(error);
```
Comment (keraf) [inline packages/suite/src/actions/suite/metadataActions.ts]: console.log here

File: packages/suite/src/actions/suite/metadataActions.ts [hunk 4]
```diff
@@ -141,9 +142,12 @@ export const disconnectProvider = () => async (dispatch: Dispatch) => {
...
     // error should be of specified type, but in case it is not (catch is not typed) show generic error
     if (!error?.code) {
         // if this happens, it means that there is a hole in error handling and it should be fixed
+        console.trace();
```
Comment (keraf) [inline packages/suite/src/actions/suite/metadataActions.ts]: and trace

File: packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx [hunk 5]
```diff
@@ -70,15 +73,40 @@ export const Target = ({
...
+                                if (!target?.addresses) {
+                                    // probably should not happen?
+                                    return addNotification({
+                                        type: 'error',
+                                        error: 'There is nothing to copy',
+                                    });
+                                }
+                                console.log('copy', target);
+                                const result = copyToClipboard(target.addresses.join(), null);
+                                console.log(result);
```
Comment (keraf) [inline packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx]: console logs here

File: packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx [hunk 6]
```diff
@@ -70,15 +73,40 @@ export const Target = ({
...
+                                if (typeof result !== 'string') {
+                                    return addNotification({ type: 'copy-to-clipboard' });
+                                }
+                                return addNotification({ type: 'error', error: result });
```
Comment (keraf) [inline packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx]: I would probably invert this here for clarity, return errors first and then get to the end where you return the actual result.

File: packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx [hunk 7]
```diff
@@ -70,15 +73,40 @@ export const Target = ({
 }: TargetProps) => {
     const targetAmount = getTargetAmount(target, transaction);
     const operation = getTxOperation(transaction);
+    // const htmlElement = createRef<HTMLDivElement>();
+    const { addNotification } = useActions({ addNotification: notificationActions.addToast });
 
     return (
         <BaseTargetLayout
+            // ref={htmlElement}
```
Comment (keraf) [inline packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx]: Can be removed (and commented declaration above too) if it's not needed anymore.

File: packages/suite/src/actions/suite/metadataActions.ts [hunk 8]
```diff
@@ -315,13 +333,18 @@ export const fetchMetadata = (deviceState: string) => async (
     try {
         await Promise.all(promises);
         // if interval for watching provider is not set, create it
-        if (!fetchInterval) {
-            fetchInterval = setInterval(() => {
-                if (!device?.state) return;
-                dispatch(fetchMetadata(device?.state));
+
+        // todo: ?? bugs here !!
```
Comment (keraf) [inline packages/suite/src/actions/suite/metadataActions.ts]: Bugs here?! :scream:

File: packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx [hunk 9]
```diff
@@ -18,88 +21,179 @@ const LabelDefaultValue = styled.div`
...
+
+    return <>{props.defaultVisibleValue}</>;
+};
+
 const getLocalizedActions = (type: MetadataAddPayload['type']) => {
     const defaultMessages = {
         add: <Translation id="TR_LABELING_ADD_LABEL" />,
-        edit: <Translation id="TR_LABELING_EDIT_LABEL" />,
+        edit: <Translation id="TR_LABELING_EDITED_LABEL" />,
+        edited: <Translation id="TR_LABELING_REMOVE_LABEL" />,
```
Comment (keraf) [inline packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx]: edit translation was changed to edited and edited is remove, is that expected?

**Review by keraf — CHANGES_REQUESTED** [review]

File: packages/integration-tests/projects/suite-web/plugins/dropbox.js (hunk 1 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/plugins/dropbox.js]: :D true

File: packages/integration-tests/projects/suite-web/tests/metadata/dropbox.test.ts (hunk 2 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/tests/metadata/dropbox.test.ts]: ah, I am removing this test. It is not need. I used it only for initial testing of the mock.

File: packages/suite/src/actions/suite/metadataActions.ts (hunk 8 above)
Comment (mroz22) [reply packages/suite/src/actions/suite/metadataActions.ts]: hah, that should be already fixed. it is the problem I was talking about. incorrectly set up intervals.

File: packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx (hunk 9 above)
Comment (mroz22) [reply packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx]: good catch, thanks

**Review by keraf — APPROVED** [review]

**Pull request merged**